### DISCUSSION TITLE
Timeline for next official release

### DISCUSSION THREAD

**Discussion by mil-ad** [ahrm/sioyek#689] [General]
Hey @ahrm, I was wondering if you're planning to make a release anytime soon? I can use sioyek-git from AUR on my own computer to be on the latest but on my shitty work Mac I rely on brew which is still on 2.0.0. 😢

**Comment by ahrm** [comment]
Currently I am working on porting sioyek to mobile devices. The next release will be when that is done. I would say most of the work is done but it is hard for me to give a timeline as there might be unexpected hiccups.
